### ISSUE TITLE
abs as an option pull request?

### ISSUE THREAD

**Issue by chrisallick** [tmlee/time_difference#21]
Hey, there is a pull request for absolution value being an option. Would love to see that get merged in. I think the continuous integration is failing. Can someone look into that?

Thanks!
Chris Allick

**Comment by moveson** [comment]
I agree that absolute value should be an option. I've incorporated the gem into my code and modified it because I need the ability to work with negative values.

**Comment by coderhs** [comment]
Ya it's 2020, I would love to see that in the Gem and this is such a useful gem and would love to see it give the info if it's negative or not.
